features/user_can_open_post.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Open up posts
+  "As a user,
+  In order to interact with a post
+  I would like to be able to open it."
+
+  Background:
+    Given the following post exists
+    | title        | text          | user_name    |
+    | Codeacademy  | Smooth-ride   | hackerman    |
```

Comment:
Please add another Post here, you are picking a post from a index

Reply:
Fixed